composer.json

```diff
@@ -13,10 +13,16 @@
             "homepage": "https://github.com/ddeboer/imap/graphs/contributors"
         }
     ],
+    "repositories": [
+    	{
+            "type": "git",
+            "url": "https://github.com/apburton84/transcoder.git"
+        }	
+    ],
     "require": {
         "php": ">=5.4.0",
         "ext-imap": "*",
-        "ddeboer/transcoder": "~1"
+        "apburton84/transcoder": "dev-master"
```

Comment:
Why `dev-master`? There appears to be a `1.0.0` release: https://packagist.org/packages/ddeboer/transcoder#1.0.0